internal/operators/manager_test.go

```diff
@@ -372,11 +372,27 @@ var _ = Describe("Operators manager", func() {
 			err := manager.EnsureOperatorArchCapability(cluster, cluster.CPUArchitecture, monitoredOperators)
 			Expect(err).To(Not(BeNil()))
 		})
-		It("error on LVM with CNV architecture", func() {
+		It("error on LVM with CNV architecture below version 4.14", func() {
```

Comment:
I don't understand how lvm is related to this test. It only checks that CNV is support with ARM.

Reply:
The operator manager has also a validation check.
ARM enablement  change also effects it's so we had to modify the unit-test 
https://github.com/openshift/assisted-service/blob/master/internal/operators/manager.go#L445C21-L445C49